rfc/rfc006-distributed-registry.md

```diff
@@ -97,11 +97,17 @@ The public key MUST also be present in the `verificationMethods` and referenced
 
 DID documents are enclosed in a message envelope to ensure consistency in the network.
 The envelope is in the form of a JWS as described in [RFC004](rfc004-verifiable-transactional-graph.md).
-Once the network layer has confirmed the signature of the JWS, the registry MUST validate if the submitter is authorized to create, update or delete the document.
-If the authorization fails, the document should be ignored.
+Once the network layer has confirmed the correctness of the signature of the JWS, the verifiable data registry MUST validate if the 
+submitter was authorized to create, update or deactivate the document. If the authorization fails, the document MUST NOT be processed by the registry.
 
-The `controller` field MAY be present. If no `controller` field is present, the DID subject itself is the controller.
-If the `controller` field is present, only the DID subjects from the `controller` field can change the DID document.
+A DID document can ony be created by its subject. To prove ownership of the private key, the JWS MUST be signed by a new private key.
```

Comment:
ony -> only

> To prove ownership of the private key, the JWS MUST be signed by a new private key.

There's no mention of "private key" above, so I think the principle should be introduced first. Something like "A new DID is created by generating a key pair and proving control over the private key by signing the transaction containing the DID document with it (the newly generated private key)".